pkg/internal/apis/acme/types_issuer.go

```diff
@@ -41,6 +41,13 @@ type ACMEIssuer struct {
 	// Only ACME v2 endpoints (i.e. RFC 8555) are supported.
 	Server string
 
+	// PreferredChain is the chain to use if the ACME server outputs multiple.
+	// PreferredChain is no guarantee that this one gets delivered by the ACME
+	// endpoint.
+	// For example, for Let's Encrypt's DST crosssign you would use:
+	// "DST Root CA X3" or "ISRG Root X1" for the newer Let's Encrypt root CA.
+	PreferredChain string `json:"preferredChain"`
```

Comment:
Should there be some validation to ensure that this meets the same requirements as commonName?

Reply:
Good idea!

Reply:
Added an openapi validation